vuepress/docs/next/docs/getting-started/entando-cli.md

```diff
@@ -0,0 +1,131 @@
+---
+sidebarDepth: 1
+---
+
+# Entando CLI
+
+The Entando Command Line Interface (CLI), **ent**, provides a set of commands that accelerate common tasks such as installing a new instance of Entando, building projects, creating and deploying bundles, and composing Entando Applications. This document covers the install, command overview, and a few operations to start using the ent tool.
```

Comment:
I removed the CLI in parenthesis. I agree it's a formality and maybe this audience doesn't need it. I didn't use the parenthesis for ent because I think it makes it less important. In this situation, I wanted to emphasize it. Looking at the kubectl page, they emphasized kubectl.